README.md

```diff
@@ -1 +1,310 @@
+```
+
+Run the docker container executing the following command:
+
+```bash
+docker run -it \
+    --net=host \
+    --ipc=host \
+    --privileged \
+    amlip
+```
+
+## Execution
+
+### Backend Server
+
+To run the AML-IP nodes and the backend server, follow the steps below:
+
+Navigate to the [backend](backend) directory
+
+```bash
+cd backend/
+```
+
+**Run the server**
+
+1. Load the AML-IP environment
+
+	```bash
+	source /AML-IP/install/setup.bash
+	```
+
+2. Start the [server](backend/server.py)
+
+	```bash
+	python3 server.py
+	```
+
+**Run AML-IP nodes**
+
+Depending on your requirements, you can run different types of AML-IP nodes:
+
+* **Run a Computing Node**
+
+	1. Load the AML-IP environment
+
+		```bash
+		source /AML-IP/install/setup.bash
+		```
+
+	2. Start the [Computing Node](backend/computing.py)
+
+		```bash
+		python3 computing.py
+		```
+
+	You can run multiple computing nodes simultaneously. Each computing node will continuously await job assignments. When running multiple nodes, they will collectively distribute the workload, optimizing the time to find a solution.
+
+	To stop a computing node, simply run `ctrl+C`.
+
+* **Run a Model Manager Sender Node**
+
+	1. Load the AML-IP environment
+
+		```bash
+		source /AML-IP/install/setup.bash
+		```
+
+	2. Start the [Model Manager Sender Node](backend/sender.py)
+
+		```bash
+		python3 sender.py
+		```
+
+	To stop a model manager sender node, simply run `ctrl+C`.
+
+* **Run an Inference Node**
+
+	1. Load the AML-IP environment
+
+		```bash
+		source /AML-IP/install/setup.bash
+		```
+
+	2. Start the [Inference Node](backend/inference.py)
+
+		```bash
+		python3 inference.py
+		```
+
+	To stop an inference node, simply run `ctrl+C`.
+
+### Frontend Marcelle Framework
+
+To run the frontend Marcelle framework, follow the steps below:
+
+**Launch the dashboard**
+
+1. Navigate to the [*frontend/aml_dashboard*](frontend/aml_dashboard) directory
+
+	```bash
+	cd frontend/aml_dashboard
+	```
+
+2. Run the dashboard
+
+	```bash
+	npm run dev
+	```
+
+With these steps completed, you're all set to utilize the AML-Dashboard, seamlessly integrating backend server functionalities with the Marcelle framework frontend.
+
+
+## Usage
+
+With the AML-Dashboard up and running, you can access it at the following address:
+
+```bash
+http://localhost:5173/
+```
+
+### Data Management Tab
+
+Efficiently manage and create new datasets.
+
+To create a dataset, simply follow these steps:
+
+1. Activate the `video` switch in the *webcam* section to enable webcam capture.
+2. Specify the label of the dataset in the *Instance label* section.
+3. Click on the `Hold to record instances` button in the *Capture instances to the training set* to start recording instances.
+4. Once recorded, the dataset will be promptly displayed in the *dataset browser* section for easy access and management.
+
+<div align="center">
+  <img src=".figures/aml-dashboard_data_management.png" width="750">
```

Comment:
I think you have uploaded the incorrect image. The one provided is from the Context Broker Tab, not the Data Management Tab.

Reply:
Done, sorry for the confusion

Reply:
No problem!